src/marine_eov_broker/MarineRiBroker.py

```diff
@@ -138,61 +138,68 @@ def build_vocabularies(self, eov) -> dict:
     
     def query_vocabularies(self, eov) -> dict:
         """
-        Query vocabulary server and search for P01 parameters names corresponding to a A05 EOV term.
-        
+        Query vocabulary server and search for P01 parameters names corresponding
+            to a A05 EOV term.
+
         Arguments :
         eov: variable string
         """
-        sparql = SPARQLWrapper(self.vocabularies_server)
+        kgsource = KGSource.build(self.vocabularies_server)
+        query_result = kgsource.query(DEFAULT_QUERY_STRINGS[eov])
+        return query_result._data
```

Comment:
`_data` ?? points to a guaranteed internal dict build from either rdflib or sqlwrapper source I presume?

however ties in with the dict based `QueryResult` implementation?
maybe cleaner to have/use some `to_dict()` on the `QueryResult`?